tools/tokenserver/loadtests/mock-fxa-server/main.py

```diff
@@ -1,20 +0,0 @@
-import functions_framework
```

Comment:
We don't need this file anymore now that we don't use a mock OAuth service for load testing